project/Dependencies.scala

```diff
@@ -5,9 +5,9 @@ object Dependencies {
 
   val CronBuild = sys.env.get("GITHUB_EVENT_NAME").contains("schedule")
 
-  val Scala213 = "2.13.10" // update even in link-validator.conf
-  val Scala212 = "2.12.17"
-  val Scala3 = "3.2.2"
+  val Scala213 = "2.13.11" // update even in link-validator.conf
+  val Scala212 = "2.12.18"
+  val Scala3 = "3.3.0"
```

Comment:
The Scala 3 version is one of those that we should align across all Akka modules. We can make this change but should be aware of that it means that downstream, such as the Cassandra plugin, must also be updated before using this version of Alpakka. Created an issue https://github.com/akka/akka/issues/32093